pulp_rpm/app/models.py

```diff
@@ -100,6 +100,8 @@ class Package(Content):
             First byte of the header
         rpm_header_end (BigInteger):
             Last byte of the header
+        rpm_modular (Bool):
+            Package is modular
```

Comment:
maybe `Flag to identify if the package is modular`?

Reply:
same as comment above